test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12822,6 +12822,10 @@ def __init__(self, name=None, attrs={}, pbsconf_file=None, snapmap={},
                                 '$vnodedef_additive': 0,
                                 '$alps_client': alps_client,
                                 '$usecp': '*:%s %s' % (usecp, usecp)}
+        elif self.platform == 'shasta':
```

Comment:
Isn't shasta also a Cray system? The condition block above this says "if platform == cray". Should 'cray' in PTL be renamed to something more specific?

Reply:
There is a plan to change the usages of "cray" to "crayxc" or "crayxt". This will also change "craysim" to "crayxcsim".
That change will also change "shasta" to "crayshasta".
We decided not to do this now as it could take a long time to change all the usages and test for regressions.